dev-aws/kafka-shared-msk/account-identity/holders.tf

```diff
@@ -1,8 +1,3 @@
-import {
```

Comment:
Is there a Terraform module configured for account-identity? As I was trying to see the last apply, but couldn't find one in here: https://terraform-applier-system.dev.aws.uw.systems/#pubsub

Some guidance on how to configure it can be found here: https://wiki.uw.systems/posts/how-to-migrate-to-amazon-msk-ztoacwsm#htsno-2-create-topics-in-msk

Reply:
https://github.com/utilitywarehouse/kubernetes-manifests/pull/95592